package.json

```diff
@@ -31,10 +31,14 @@
     "eslint": "^7.32.0",
     "eslint-config-airbnb": "^19.0.4",
     "eslint-config-airbnb-typescript": "^17.0.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-prefer-arrow": "^1.2.3",
+    "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-rulesdir": "^0.2.1",
     "eslint-plugin-sonarjs": "^0.16.0",
     "eslint-webpack-plugin": "^3.2.0",
     "gatsby-plugin-eslint": "^4.0.2",
+    "prettier": "2.7.1",
```

Comment:
I see no script, how do you use it?

Reply:
I'm using it in my VS code editor: https://github.com/prettier/prettier-vscode, whenever I need to format a code. I understand your point, it is possible to keep it away from our repo I don't have a strong opinion on it, but it is recommended by prettier docs https://prettier.io/docs/en/install.html to have it installed locally. I can add the following script with `.prettierignore` to our repo:
```sh
prettier --write .
```
Let me know

Reply:
I see. Can you add something like `"format": "prettier --write ."` to `scripts` then?

Reply:
Sure, I've added it to the `scripts`.